lib/read.js

```diff
@@ -9,6 +9,8 @@ let db = require('./db');
 let FilterCompiler = require('./filter');
 let Promise = require('bluebird');
 
+const VALDID_START_DATE_MIN = new Date(1000000000000).getTime();
```

Comment:
What's a `VALDID`? Also `new Date(1000000000000).getTime()` is just `1000000000000` so you don't need the Date stuff at all.